Clarify OpenShift pseudo-check documentation (#5596)
DataDog/integrations-core: openshift/README.md

#### Custom Datadog SCC for all features

If SELinux is in permissive mode or disabled, enable the `hostaccess` SCC to benefit from all features.

> If SELinux is in enforcing mode, it is recommend to grant [the `spc_t` type][7] to the datadog-agent pod. In order to deploy our agent, Datadog created a [datadog-agent SCC][8] that you can apply after [creating the datadog-agent service account][5]. It grants the following permissions:

Reviewer: ```suggestion
If SELinux is in enforcing mode, it is recommended to grant [the `spc_t` type][7] to the datadog-agent pod. In order to deploy the agent you can use the following [datadog-agent SCC][8] that can be applied after [creating the datadog-agent service account][5]. It grants the following permissions:
```
